contracts/src/Lightning.ts

```diff
@@ -260,3 +270,13 @@ export class Lightning extends SmartContract {
     ]);
   }
 }
+
+export class LightningTokenHoder extends SmartContract {
```

Comment:
What's the purpose of `LightningTokenHolder`?

Reply:
It is how the custom token works in zkApp. It is to attach the token update for Lightning contract under Example token. See https://docs2-git-tutorial-11-minadocs.vercel.app/zkapps/tutorials/advanced-account-updates#zkapp-token-management to grasp why is it needed. To my understanding, as zkApp isn't allowed to use map structure, Mina chose to support the custom token facility natively, which made such ownership relations to be formidable.